platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileManager.java

```diff
@@ -318,9 +318,13 @@ private Path getSignedStateDir(final long round) {
      * @param signedState       the state in question
      * @param previousTimestamp the timestamp of the previous state that was saved to disk, or null if no previous state
      *                          was saved to disk
+     * @param source            the source of the signed state
      * @return true if the state should be written to disk
      */
-    private boolean shouldSaveToDisk(final SignedState signedState, final Instant previousTimestamp) {
+    private boolean shouldSaveToDisk(
```

Comment:
If I understand the data flow correctly, changes to this class can be reverted if you make my previously suggested change.

Reply:
I don't think I agree with the change, we can discuss if you like